diff --git a/browser/base/content/aboutNetError.js b/browser/base/content/aboutNetError.js
--- a/browser/base/content/aboutNetError.js
+++ b/browser/base/content/aboutNetError.js
@@ -766,7 +766,7 @@
     // Also don't bother if it's just the user's clock being off...
     !clockSkew &&
     // Symantec distrust is likely harmless as well.
-    failedCertInfo.erroCodeString !=
+    failedCertInfo.errorCodeString !=
       "MOZILLA_PKIX_ERROR_ADDITIONAL_POLICY_CONSTRAINT_FAILED"
   ) {
     document.body.classList.add("caution");
